routes/articles.js

```diff
@@ -9,26 +9,45 @@ router.post('/', async (req,res) => {
         title : req.body.title,
         description :  req.body.description,
         author : req.body.author,
-        tags :  req.body.tags
+        tags :  req.body.tags,
+        time : req.body.time
     })
     try {
         const newArticle = await article.save()
         //201 cause its sucess
         res.status(201).json(newArticle)
```

Comment:
should it follow the same convention as
`res.status(401).json( { message : err.message })`

I mean something like this -> res.status(201).json({message:newArticle})

why 

In front end if somone does 
```
if(res.message == "error")
refresh()
```

but this will throw an error since message is undefined in frontend So to be consistent let's keep same format

Reply:
Sure will do that asap.